src/keyboard/behavior-short-names.json

```
@@ -0,0 +1,14 @@
+{
+    "Key Press": {"short": ""},
+    "Bootloader": {"short": "Bootloadr"},
+    "External Power": {"short": "Ext Pwr"},
+    "Grave/Escape": {"short": "Grv/Esc"},
+    "Key Repeat": {"short": "Key Rept"},
+    "Key Toggle": {"short": "Key Togg"},
+    "Momentary Layer": {"short": "MomntLayr"},
```

Not sure I love this one.... Maybe:

```suggestion
    "Momentary Layer": {"short": "MLayer"},
```